csun-metalab/nomi
# NAM-457 -- UI: Make the semester, term year and feedback button disappear from the navbar when you are logged out.

# Description
Applied nav-bar component to the main views instead of the index.
  
# Testing
1. Ensure the login bar appears like normal.
2. Check every main component and ensure all of them contain the nav-bar component.
3. Check to see if all functionality of the nav-bar should work like it should.
